tests/Table/TrTest.php

```
@@ -0,0 +1,879 @@
+            'Failed asserting that user-defined attributes override global defaults correctly.',
+        );
+
+        SimpleFactory::setDefaults(
+            Tr::class,
+            [],
+        );
+    }
+
+    public function testReturnNewInstanceWhenSettingAttribute(): void
+    {
+        $tr = Tr::tag();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Table\Tr' in method 'testReturnNewInstanceWhenSettingAttribute'. (undefined)

(StaticAccess)